docs/options.md

```diff
@@ -915,10 +917,12 @@ Platform-specific environment variables are also available:<br/>
     repair-wheel-command = "auditwheel repair --lib-sdir . -w {dest_dir} {wheel}"
 
     # Multi-line example
+    # Use abi3audit to catch issues with Limited API wheels
     [tool.cibuildwheel]
     repair-wheel-command = [
       'python scripts/repair_wheel.py -w {dest_dir} {wheel}',
       'python scripts/check_repaired_wheel.py -w {dest_dir} {wheel}',
+      'pipx run abi3audit --strict --report {wheel}',
     ]
```

Comment:
Adding to this example is perhaps a little confusing - I think we should show an other multiline example that extends the default, something like

```
    # Use abi3audit to catch issues with Limited API wheels
    [tool.cibuildwheel.linux]
    repair-wheel-command = [
      "pipx run abi3audit --strict --report {wheel}",
      "auditwheel repair -w {dest_dir} {wheel}",
    ]
    [tool.cibuildwheel.macos]
    repair-wheel-command = [
      "pipx run abi3audit --strict --report {wheel}",
      "delocate-wheel --require-archs {delocate_archs} -w {dest_dir} -v {wheel}",
    ]
    [tool.cibuildwheel.windows]
    repair-wheel-command = "pipx run abi3audit --strict --report {wheel}"
```